get-started/upload-images.md

```diff
@@ -37,7 +37,7 @@ tinymce.activeEditor.uploadImages(function(success) {
 
 In order to upload local images to the remote server, you will need a server-side upload handler script that accepts the images on the server, stores them appropriately, and returns a JSON object containing the location that they were uploaded to.
 
-An example PHP upload handler implementation is available [here](../php-upload-handler/).
+An example PHP upload handler implementation is available [here](../../advanced/php-upload-handler/).
```

Comment:
hi @EricHogue thanks for the PR, greatly appreciated (!) and apologies for the delay in getting to this.

By my reckoning we don't need `../..` and so `/advanced/php-upload-handler/` should do the trick as the build process interprets this url as `{baseurl}/advanced/...` i.e. `tinymce.com/docs/advanced/php-upload-handler`.

In case you're interested, `../advanced/php-upload-handler/` would be interpreted as `{baseurl}[parent_folder]/advanced/...` i.e. `tinymce.com/docs/get-started/advanced/php-upload-handler`.

And for anyone else reading this, the issue Eric fixed involved the build process converting `../php-upload-handler/`  to `tinymce.com/get-started/php-upload-handler/` returning a 404.

Eric, let me know how you want to proceed: you making the change or me. Cheers!